gateway-ha/src/main/java/io/trino/gateway/ha/module/ClusterStateListenerModule.java

```diff
@@ -53,7 +52,6 @@ public List<TrinoClusterStatsObserver> getClusterStatsObservers(
             BackendStateManager backendStateManager)
     {
         observers = new ArrayList<>();
-        NotifierConfiguration notifierConfiguration = getConfiguration().getNotifier();
```

Comment:
So notifierConfiugration is never access or used? And same for all the others?

Reply:
NotifierConfiguration is still used from NotifierModule.

Reply:
Sure .. but no in this class .. hence the removal. And the same logic applies to other removals.. right?